repos/system_upgrade/el7toel8/actors/storagescanner/libraries/library.py

```diff
@@ -79,22 +80,41 @@ def _get_fstab_info(fstab_path):
     ''' Collect storage info from /etc/fstab file '''
     if _is_file_readable(fstab_path):
         with open(fstab_path, 'r') as fstab:
-            for entry in fstab:
+            keys = ['fs_spec', 'fs_file', 'fs_vfstype', 'fs_mntops', 'fs_freq', 'fs_passno']
+            for line, entry in enumerate(fstab, 1):
                 if entry.startswith('#'):
                     continue
 
+                if '#' in entry:
+                    message = "Your /etc/fstab has '#' comment in line {} please verify" \
+                        "if it's correct.".format(line)
+                    api.current_logger().warning(message)
```

Comment:
You should also strip the comment from the line else you can get inhibitor for valid fstab when it has comment at the end or not get it when there is comment somewhere in the middle but some options are missing

Reply:
I did it before but @pirat89  has commented about it, the '#' is a valid character when you have a password in options for example.

Now we are warning the user when there is the '#'